code/game/objects/items/plushes.dm

```diff
@@ -473,18 +473,25 @@
 	icon_state = "narplush"
 	divine = TRUE
 	var/clashing
-	var/is_invoker = TRUE
+	var/invoker_charges = 2
 	gender = FEMALE	//it's canon if the toy is
 
+/obj/item/toy/plush/narplush/examine(mob/user)
+	. = ..()
+	if(invoker_charges == 0)
```

Comment:
This can be used for metaknowledge since it's not with `is_cultist()`

Reply:
Looks like the hugbox variant isn't even used at all
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/30 960302/c3b66c4a-95f3-4428-a1d1-d8e34e0c254d)

The only form of "meta knowledge" that can be used with the plush is if the cult uses it twice already and leaves the plush lying around, which in that case, they kind of deserve it as they are leaving evidence of their wrongdoing lying around.

Reply:
No, I am talking when a plushie reports `'it looks tired'`, it's a thing that might need to be contrabanded or burnt in chappel, because you can recognise it has charges by it.